src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -134,19 +135,16 @@ public function showAction(Post $post)
      * @Route("/{id}/edit", requirements={"id": "\d+"}, name="admin_post_edit")
      * @Method({"GET", "POST"})
      */
-    public function editAction(Post $post, Request $request)
+    public function editAction(Request $request, EntityManagerInterface $em, Post $post, Slugger $slugger)
```

Comment:
I see Doctrine as a regular dependency so I would put it at the end. 
Anyway we can decide this later in https://github.com/symfony/symfony-docs/pull/7933#discussion_r117676576, let's keep it as is for now :)